src/setup-gcloud.ts

```diff
@@ -47,6 +49,15 @@ export async function run(): Promise<void> {
       core.addPath(path.join(toolPath, 'bin'));
     }
 
+    // Install additional components
+    const components = core.getInput('install_components');
+    if (components) {
+      await installComponent(
+        components.split(',').map((comp) => comp.trim()),
+        false,
```

Comment:
Any reason why we are setting silent false explicitly? I noticed for other setupGcloudSDK commands we default to true.

Reply:
This was for debugging. silent = false will print command output to the console.